app/api/utils.py

```diff
@@ -55,3 +57,44 @@ def cli_for_function(function, outfile, url):
         processed_df.to_csv(outfile, index=False)
     else:  # print to STDOUT
         print(processed_df.to_csv(index=False))
+
+def get_all_state_finals():
+    states_docs_urls = pd.read_csv("app/api/state_docs_urls.csv")
+    return states_docs_urls['Finals'].tolist()
+
+def get_all_priortize_entries():
+    entries, finals = [], []
+    for _, state_row in states_docs_urls.iterrows():
+        if state_row['Entry']:
+            entries.append(state_row['Entry'])
+        else:
+            finals.append(state_row['Final'])
+
+    return (entries, finals)
+
+def run_function_on_states(function, entries, finals, outputDir):
+    """
+    Call the function on every google sheets url for the specified states.
```

Comment:
Could you make a note that entries and finals are lists of state abbreviations?

Reply:
Sure